react-game/src/components/AllTrials.js

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import StartButton from './Buttons/StartButton';
+import classes from './AllTrials.module.css';
+import TrialRow from './TrialRow/TrialRow';
+
+function AllTrials() {
```

Comment:
please use arrow function instead of normal function to define react components

Reply:
done